modules/builds-strategy-custom-expose-docker-socket.adoc

```diff
@@ -18,3 +18,11 @@ strategy:
   customStrategy:
     exposeDockerSocket: true
 ----
+
+[NOTE]
+====
+Setting this value simply causes the build pod to attempt to mount `/var/run/docker.sock` from the
+node host.  If your cluster's node hosts do not have a docker daemon, your builds will not be able
+to leverage it.  In this case, consider using `buildah` to perform image build/push operations
+instead.  See the <custom build tutorial> for an example of how this can be done.
+====
```

Comment:
in lieu of this note, i think we should just drop this "expose-docker-socket" module entirely, and mention in the "removed functionality" section that mounting the docker socket in custom builds will not work on hosts that do not have a docker daemon which is all hosts in 4.0.